src/components/Repository/Repository.test.tsx

```diff
@@ -19,7 +20,7 @@ const mockPackageMeta: jest.Mock = jest.fn(() => ({
 }));
 
 jest.mock('../../pages/Version', () => ({
-  DetailContextConsumer: component => {
+  DetailContextConsumer: (component: { children: (arg0: { packageMeta: PackageMetaInterface }) => void }) => {
```

Comment:
sorry, but I am not so sure I like it...maybe there is a simpler way?

Reply:
It was auto generated by vscode, it can now generate types based on the usage 🚀 
Didn't think it would work but it worked.
I can properly type this if wanted.

Reply:
Hm... I refactored this component today... I've removed this part of the code, so that is fine :-)

Reply:
Well but then we have to resolve the conflicts now ;-)